### ISSUE TITLE
Template pack library

### ISSUE THREAD

**Issue by tiberiuana** [pennersr/django-allauth#1929]
Hello all,

Noticing that I was repeating myself and copy pasting code among projects, I've published a small sidekick library that provides standalone, drop-in templates for allauth. I started with Bootstrap 4, and intend to add templates for other major front end frameworks too, to match what's used in the main app.

[dead link]

Any comments and contributions are appreciated. Thanks!

**Comment by tiberiuana** [comment]
...and, sure enough, the feedback I got so far was that it's not a good idea to clutter the virtualenv with lots of static files, especially if they're unused.

I've killed the previously linked project and separated the templates alone in an even smaller package.

https://github.com/tiberiuana/django-allauth-templates-bootstrap4

If anyone knows that I'm repeating existing work, I'd love a link.to another project I could be contributing instead.
